pkgs/applications/misc/zettlr/generic.nix

```diff
@@ -19,21 +21,27 @@ in
 appimageTools.wrapType2 rec {
   inherit pname version src;
 
-  extraPkgs = pkgs: [ pkgs.texliveMedium pkgs.pandoc ];
+  extraPkgs = pkgs: [
+    pkgs.texliveMedium
+    pkgs.pandoc
+  ];
 
   extraInstallCommands = ''
+    source "${makeWrapper}/nix-support/setup-hook"
+    wrapProgram $out/bin/zettlr \
```

Comment:
Is it being wrapped two times?

Reply:
Ah, I'm not sure as most AppImages have the setup hook and wrapProgram.

Reply:
`ls -la ./result/bin` ?

Reply:
Is this normal?
```
total 16
dr-xr-xr-x 2 root root 4096 Dec 31  1969 .
dr-xr-xr-x 4 root root 4096 Dec 31  1969 ..
-r-xr-xr-x 1 root root  279 Dec 31  1969 zettlr
lrwxrwxrwx 1 root root   62 Dec 31  1969 .zettlr-wrapped -> /nix/store/igi5gk4k3nyszmw53rsn2pfhwnxj85c4-zettlr-3.2.0-bwrap
```